services/QuillLMS/engines/evidence/app/controllers/evidence/automl_models_controller.rb

```diff
@@ -53,6 +53,22 @@ def deployed_model_names
       render json: @names
     end
 
+    def enable_more_than_ten_labels
+      prompt_id = params['prompt_id']
+      automl_model = Evidence::AutomlModel.where(prompt_id:).last
+
+      return render json: { error: "Model not found with prompt id #{prompt_id}" }, status: 404 if automl_model.nil?
+
+      parsed_additional_labels = params['additional_labels'].split(',').map(&:strip)
+      labels = (automl_model.labels + parsed_additional_labels).sort.uniq
```

Comment:
Nice job doing a unique check here to prevent duplicates. I was checking for this.